config/development.js

```diff
@@ -21,6 +21,8 @@ module.exports = {
 
   enableStrictMode: true,
 
+  // I had to add this to get it to work, but I think there's a better way.
+  proxyPort: 3000,
```

Comment:
no, I think it's fine because when we remove `NODE_APP_INSTANCE`, node-config will stop loading the `*-amo.js` files (assuming `NODE_APP_INSTANCE` is set to `amo`.

Reply:
Ok, and thanks for pointing me to the source of how multiple config files are being loaded. :-)

Given that we're no longer loading any of the `-amo` files anymore, do you think we should include merging those in this patch? I'm a bit worried that something other than just `proxyPort` is going to be missing with this patch if we don't also merge the files.

WDYT?

Reply:
Yeah, we should definitely get rid of the `-amo` config files first. Maybe we could start with a patch only for that, and then we apply this patch on top?

Merging the configs is going to be tricky because we have to do it right :p I'd personally open a PR only for that. If `NODE_APP_INSTANCE` is set and there is no specific config, I suppose it wouldn't complain. (If it does, we can have empty files)

Reply:
That's a good idea. I'll open an issue for that and put this on hold for now.